### PR TITLE
Add sort by serial number in the profile collection

### LINKED ISSUE #1305: A weird organisation of the nfts in a collection
### Please reproduce in steps

https://nft.kodadot.xyz/rmrk/collection/FC77C33AB229A2056A-BTFLSUZANN

------------
its should be from 1 to 30: 
1 - 2 - 3 - 4 - 5 - 6 - 7 - 8 - 9 - 10 - 11 - 12 - 13 - 14 - 15 - 16 - 17 - 18 - 19 - 20 - 21 - 22 - 23 - 24 - 25 - 26 - 27 - 28 - 29 - 30 
but its going like this:
1 -  10 - 11 - 12 - 13 - 14 - 15 - 16 - 17 - 18 - 19 - 2 - 20 - 21 - 22 - 23 - 24 - 25 - 26 - 27 - 28 - 29 -  3 - 30 - 4 - 5 - 6 - 7 - 8 - 9 

### Expected Behavior

its should be from 1 to 30: 
1 - 2 - 3 - 4 - 5 - 6 - 7 - 8 - 9 - 10 - 11 - 12 - 13 - 14 - 15 - 16 - 17 - 18 - 19 - 20 - 21 - 22 - 23 - 24 - 25 - 26 - 27 - 28 - 29 - 30

### What browsers are you seeing the problem on?

Chrome

### Are you logged in?

Yes

### At which address did you encounter bug?

nft.kodadot.xyz / kodadot.xyz

### Screenshots

/

### Relevant log output

```shell
/
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #1305 
- [x] add sort by serial number in the profile collection
- [x] adjust text size on the small display

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged the recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://user-images.githubusercontent.com/734428/161207192-b82fdb3a-fc7b-4305-8478-4b9bf9aaa6a9.mov

### CODE REVIEW

File: langDir/en.json [hunk 1]
```diff
@@ -418,6 +418,7 @@
     "UPDATED_AT_ASC": "Least Interacted",
     "PRICE_DESC": "Price: High to Low",
     "PRICE_ASC": "Price: Low to High",
+    "SN_ASC": "Serial Number",
```
Comment (vikiival) [inline langDir/en.json:421]: Serial Number ascending

File: components/rmrk/Gallery/GalleryCard.vue [hunk 2]
```diff
@@ -111,6 +112,17 @@ export default class GalleryCard extends mixins(AuthMixin) {
...
+  get layoutClass() {
+    if (
+      this.$store.getters['preferences/getLayoutClass'] ===
+      'is-one-quarter-desktop is-one-third-tablet'
+    ) {
+      return ''
+    }
+
+    return 'title is-4'
+  }
```
Comment (vikiival) [inline components/rmrk/Gallery/GalleryCard.vue:125]: What's this?

**Review by vikiival — CHANGES_REQUESTED** [review]
Tested, like the solution. 
let's resolve last things 🚀

File: components/rmrk/Gallery/GalleryCard.vue (hunk 2 above)
Comment (preschian) [reply components/rmrk/Gallery/GalleryCard.vue:125]: oh, for this one, is it ok to adjust the text smaller in "Small Display" = 4 columns? if not, I will revert it

Comment (vikiival) [reply components/rmrk/Gallery/GalleryCard.vue:125]: then is suggest to use [vue style](https://v2.vuejs.org/v2/guide/class-and-style.html)

```html
<span
  :class="{ 'title is-4': smallDisplay  }"
></span

```


```ts
get smallDisplay(): boolean {
  this.$store.getters['preferences/getLayoutClass'] ===
      'is-one-quarter-desktop is-one-third-tablet'
}
```

**Review by vikiival — CHANGES_REQUESTED** [review]
cc @preschian 

I am fine with your proposal. Just need to 💅 polish it

de7


Also please fix the translation for `SN_ASC`

File: components/rmrk/Gallery/GalleryCard.vue (hunk 2 above)
Comment (preschian) [reply components/rmrk/Gallery/GalleryCard.vue:125]: oh, nice, looks cleaner
will update that

**Review by vikiival — APPROVED** [review]
🇵🇱 Looks lean. 

Nice polish lesson.

**Comment by preschian** [comment]
thank you so much for the review 🙏🏻

**Comment by vikiival** [comment]
Really nice

https://deploy-preview-2727--koda-nuxt.netlify.app/rmrk/collection/160a6f4320f11acb25-VVGOX?tab=collection&search=&sort=SN_ASC

**Comment by vikiival** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 189.17 USD/KSM ~ 0.529 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0xa1aa6779c0800fafbf0d41288d6ae825a6bf221055c4b3ba8cf76065ef9f8c67](https://kusama.subscan.io/extrinsic/0xa1aa6779c0800fafbf0d41288d6ae825a6bf221055c4b3ba8cf76065ef9f8c67)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**